src/constants.js

```diff
@@ -14,5 +14,11 @@ module.exports = {
     LEAFLET: 'leaflet'
   },
 
-  GMAPS_BASE_LAYER_TYPES: ['roadmap', 'gray_roadmap', 'dark_roadmap', 'hybrid', 'satellite', 'terrain']
+  GMAPS_BASE_LAYER_TYPES: ['roadmap', 'gray_roadmap', 'dark_roadmap', 'hybrid', 'satellite', 'terrain'],
```

Comment:
This file can become a little bit of a disaster if we keep adding stuff to it. I'd prefer to declare constants in the files that they "belong to" (eg: https://github.com/CartoDB/cartodb.js/blob/v4/src/analysis/analysis-model.js#L145). But that's just my opinion!

Reply:
Yes, I talked about this in another PR, the idea if the file starts becoming a disaster is moving it to a `constants` folder, and then have different files for each type of constants, for exampe `constants/layer-types.js`, `constants/errors.js`, `constants/widget-types.js`, etc